tecken/useradmin/admin.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+from tecken.tokens.models import Token
+from tecken.upload.models import Upload
+
+
+# Unregister the original UserAdmin and register our better one
+try:
+    admin.site.unregister(User)
+except TypeError:
+    pass
+
+
+@admin.register(User)
+class UserAdminBetter(UserAdmin):
```

Comment:
I cribbed this from Socorro. It shows users, but the bits that are helpful for us. We don't use the "username" field, so it's better to show users by email address. Further, it's nice to see number of uploads and api tokens and which groups they're in.